irc_cap.c

```diff
@@ -0,0 +1,174 @@
+/********************************************************************\
+  * BitlBee -- An IRC to other IM-networks gateway                     *
+  *                                                                    *
+  * Copyright 2002-2013 Wilmer van der Gaast and others                *
+  \********************************************************************/
+
+/* IRCv3 CAP command                                                    */
+
+/*
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License with
+  the Debian GNU/Linux distribution in /usr/share/common-licenses/GPL;
+  if not, write to the Free Software Foundation, Inc., 51 Franklin St.,
+  Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
```

Comment:
Can you add a link to the "RFC" here and maybe in other places where it'd be helpful? (For example with anchor links if possible.) I'll leave the latter to your judgement, mostly care about at least a link here.

I'll continue reading and see if I can review this without having to search for the doc myself now. :-)